src/Identity/UI/src/build/Microsoft.AspNetCore.Identity.UI.targets

```diff
@@ -0,0 +1,23 @@
+<Project>
+  <Import Project="$(MSBuildThisFileDirectory)Microsoft.AspNetCore.StaticWebAssets.targets"/>
+
+  <Target Name="ValidateIdentityUIFrameworkVersion" BeforeTargets="Build">
+
+    <PropertyGroup>
+      <_IdentityUIUnknownVersionErrorMessage>The 'IdentityUIFrameworkVersion' '$(IdentityUIFrameworkVersion)' in '$(MSBuildProjectFile)' is not valid. Valid versions are 'Bootstrap4'.
```

Comment:
Does this need to have 5 too?

Reply:
Yeah I don't think these are working at all, I tried bootstrap7 in the default samples ui and there was no errors, since the ci is still having issues, I'll see if I can get a fix for the error values in this PR, otherwise I'll file a follow up issue to track this

Reply:
Filed https://github.com/dotnet/aspnetcore/issues/36288 to follow up with the validation not working